hack/lib/build/binaries.sh

```diff
@@ -219,6 +219,61 @@ os::build::internal::build_binaries() {
         local_ldflags+=" -s"
       fi
 
+      #Add Windows File Properties/Version Info and Icon Resource for oc.exe
+      if [[ "$platform" == "windows/amd64" ]]; then
+        os::build::version::get_vars
+        os::util::ensure::gopath_binary_exists 'goversioninfo'
+        local major="${OS_GIT_MAJOR}"
+        local minor="${OS_GIT_MINOR%+}"
+        local patch="${OS_GIT_PATCH}"
+        local windows_versioninfo_file=`mktemp --suffix=".versioninfo.json"`
+        cat <<EOF >"${windows_versioninfo_file}"
+{
+	"FixedFileInfo":
+	{
+		"FileVersion": {
+			"Major": ${major},
+			"Minor": ${minor},
+			"Patch": ${patch}
+		},
+		"ProductVersion": {
+			"Major": ${major},
+			"Minor": ${minor},
+			"Patch": ${patch}
+		},
+		"FileFlagsMask": "3f",
+		"FileFlags ": "00",
+		"FileOS": "040004",
+		"FileType": "01",
+		"FileSubType": "00"
+	},
+	"StringFileInfo":
+	{
+		"Comments": "",
+		"CompanyName": "Red Hat, Inc.",
+		"InternalName": "openshift client",
+		"FileVersion": "${OS_GIT_VERSION}",
+		"InternalName": "oc",
+		"LegalCopyright": "© Red Hat, Inc. Licensed under the Apache License, Version 2.0",
+		"LegalTrademarks": "",
+		"OriginalFilename": "oc.exe",
+		"PrivateBuild": "",
+		"ProductName": "OpenShift Client",
+		"ProductVersion": "${OS_GIT_VERSION}",
+		"SpecialBuild": ""
+	},
+	"VarFileInfo":
+	{
+		"Translation": {
+			"LangID": "0409",
+			"CharsetID": "04B0"
+		}
+	}
+}
+EOF
+        goversioninfo -o ${OS_ROOT}/cmd/oc/oc.syso ${windows_versioninfo_file} 
```

Comment:
Are you mutating a file inside of the normal source tree?  That's generally something we don't do.  Why is the file required to be in `cmd/oc`?